app/models/warehouse_report/overlapping_coc_by_project_type.rb

```diff
@@ -177,24 +177,55 @@ def details_clients
     client_num = 0
     clients_by_id = shared_clients.index_by(&:id)
     relevant_client_ids = @non_overlapping ? non_overlapping_client_ids : overlapping_client_ids
```

Comment:
Don't we usually prefer `relevant_client_ids = if ...` elsewhere? I don't mind `?:`, I just don't usually see it in this code...

Reply:
We do, that is old code I didn't cleanup yet.